editor/utils/editor-media-upload/index.js

```diff
@@ -34,6 +34,7 @@ export default function editorMediaUpload( {
 	const errorHandler = ( { file, sizeAboveLimit, generalError } ) => {
 		let errorMsg;
 		if ( sizeAboveLimit ) {
+			// translators: %s: file name
```

Comment:
You probably need to move it inside `sprintf` just before the line with `__( ... )`.